eva_cttv_pipeline/cellbase_records.py

```diff
@@ -21,7 +24,11 @@ def __iter__(self):
 
     def __get_curr_response(self):
         reader = codecs.getreader("utf-8")
-        answer = urllib.request.urlopen('http://' + config.HOST + '/cellbase/webservices/rest/v3/hsapiens/feature/clinical/all?source=clinvar&skip=' + str(self.skip) + '&limit=' + str(self.limit))
+        url = 'http://' + config.HOST + \
+              '/cellbase/webservices/rest/v3/hsapiens/' + \
+              'feature/clinical/all?source=clinvar&skip=' + \
+              str(self.skip) + '&limit=' + str(self.limit)
```

Comment:
Using string.format instead of + concatenation could improve readability and performance.